datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1839,6 +1870,36 @@ type DataPlatformInstance implements Entity {
     Edges extending from this entity
     """
     relationships(input: RelationshipsInput!): EntityRelationshipsResult
+
+    """
+    Additional read only properties associated with a data platform instance
+    """
+    properties: DataPlatformInstanceProperties
+
+    """
+    Ownership metadata of the data platform instance
+    """
+    ownership: Ownership
+
+    """
+    References to internal resources related to the data platform instance
+    """
+    institutionalMemory: InstitutionalMemory
+
+    """
+    Tags used for searching the data platform instance
+    """
+    tags: GlobalTags
```

Comment:
if we add domain and glossary terms we can also add that here!

Reply:
See comment https://github.com/datahub-project/datahub/pull/7177#discussion_r1092361062